snowchange.py

```diff
@@ -113,28 +114,39 @@ def get_all_scripts_recursively(root_directory, verbose):
   # Walk the entire directory structure recursively
   for (directory_path, directory_names, file_names) in os.walk(root_directory):
     for file_name in file_names:
+      
       file_full_path = os.path.join(directory_path, file_name)
       script_name_parts = re.search(r'^([V])(.+)__(.+)\.sql$', file_name.strip())
+      repeatable_script_name_parts = re.search(r'^([R])__(.+)\.sql$', file_name.strip())
 
-      # Only process valid change scripts
-      if script_name_parts is None:
+      # Set script type depending on whether it matches the versioned file naming format
+      if script_name_parts is not None:
+        script_type = 'V'
+        if verbose:
+          print("Versioned file " + file_full_path)
+      elif repeatable_script_name_parts is not None:
+        script_type = 'R'
+        if verbose:
+          print("Repeatable file " + file_full_path)
+      else:
         if verbose:
           print("Ignoring non-change file " + file_full_path)
         continue
-
+      
       # Add this script to our dictionary (as nested dictionary)
       script = dict()
       script['script_name'] = file_name
       script['script_full_path'] = file_full_path
-      script['script_type'] = script_name_parts.group(1)
-      script['script_version'] = script_name_parts.group(2)
-      script['script_description'] = script_name_parts.group(3).replace('_', ' ').capitalize()
+      script['script_type'] = script_type
+      script['script_version'] = None if script_type == 'R' else script_name_parts.group(2)
+      script['script_description'] = (repeatable_script_name_parts.group(2) if script_type == 'R' else script_name_parts.group(3)).replace('_', ' ').capitalize()
```

Comment:
Please apply the same script name clean-up that we use for the versioned migrations. It makes the description more readable in the logs.

Reply:
Hi @jeremiahhansen,
Thanks for reviewing the changes! 

The same script name clean-up logic (i.e. remove underscores and capitalize) has been applied to both versioned and non versioned scripts to obtain the script_description. I am referring to line 142.

Are you referring to anything else? Am I missing anything?

Thanks!
Kedar